dependencies.props

```diff
@@ -113,6 +112,7 @@
       <ElementName>NETStandardLibraryPackageVersion</ElementName>
       <PackageId>NETStandard.Library</PackageId>
     </XmlUpdateStep>
+    <!--
```

Comment:
As an aside we can likely remove that WCF dependencies now that UWP has been removed in master. https://github.com/dotnet/core-setup/commit/3d9abcca488b38f87ee59e776f98915c0c12f729.

Reply:
Cool, I'll go ahead and clean this up here.